sdk/ml/azure-ai-ml/azure/ai/ml/entities/_assets/_artifacts/_package/model_package.py

```diff
@@ -260,20 +267,21 @@ def _load(
         data: Optional[Dict] = None,
         yaml_path: Optional[Union[PathLike, str]] = None,
         params_override: Optional[list] = None,
-        **kwargs,
+        **kwargs: Any,
     ) -> "ModelPackage":
         params_override = params_override or []
         data = data or {}
         context = {
             BASE_PATH_CONTEXT_KEY: Path(yaml_path).parent if yaml_path else Path("./"),
             PARAMS_OVERRIDE_KEY: params_override,
         }
-        return load_from_dict(ModelPackageSchema, data, context, **kwargs)
+        return cast(ModelPackage, load_from_dict(ModelPackageSchema, data, context, **kwargs))
```

Comment:
Instead of casting, could we do something like this:

```python
model_pkg: ModelPackage = load_from_dict(ModelPackageSchema, data, context, **kwargs)
```

Let's avoid making actual function calls in cases where we could just declare a variable and type it.